### ISSUE TITLE
Scroll event does't appear to trigger with window.scrollTo

### ISSUE THREAD

**Issue by micahblu** [assaf/zombie#800]
I need to detect changes to the DOM after a scroll event, however `browser.document.window.scrollTo()` doesn't appear to do anything.

This is what I have:

``` javascript
var assert = require('assert'),
    Browser = require('zombie');

Browser.localhost('http://localhost:8080', 8080);

var browser = Browser.create();
it('Should load content on scroll', function(done){

    browser.visit('/', function(error){
        assert.ifError(error);
        browser.assert.success();

        // this event never triggers
        browser.document.window.addEventListener('scroll', function(){
            console.log('scrolling...');
        });

        browser.document.window.scrollTo(100, 100);

        console.log(browser.document.documentElement.scrollTop);  // outputs '0'
        console.log(browser.document.documentElement.scrollLeft); // outputs '0'

        done();
    });
});
```

p.s This thing is amazing :+1:

**Comment by ArsalanDotMe** [comment]
I'm in dire need of this scrollTo too! I'm working to test an infinite loader and I need to trigger it by scrolling at the bottom. I guess I'll browse through the source to see how I could implement it but please bump it to the priority features list!

**Comment by cmisenas** [comment]
@assaf is this implemented or somewhere down the roadmap?

echoing top comments p.s :)

**Comment by assaf** [comment]
I don't think this is implemented anywhere. This is classical open source, contributors solve whatever problem they have, and contribute it upstream if they want to. No one is working from a roadmap.

**Comment by PickleJr** [comment]
*Disclaimer: I haven't used zombie that much, I've only stumbled here from a Google Search.

I was having the same issue in Firefox Development Edition and Microsoft Edge. I'm not sure if Chrome has this issue or not. I was able to remedy the issue by calling
`window.dispatchEvent(new Event('scroll'));`
after the window.scrollTo(...) statement.

I'm not sure if this will help everybody, but it might help somebody.
